src/webgl/framebuffer.js

```diff
@@ -222,7 +222,7 @@ export default class Framebuffer extends Resource {
     this.gl.bindFramebuffer(GL_FRAMEBUFFER, this.handle);
 
     if (color || depth || stencil) {
-      clear({color, depth, stencil});
+      clear(this.gl, {color, depth, stencil});
```

Comment:
This handles the crash which was the primary objective of this PR.

The line on 232 worries me though `this.gl.bindFramebuffer(GL_FRAMEBUFFER, null);` - what if another framebuffer was bound? 

It will be a separate PR but we absolutely need a solution for this to shore up our framebuffer API. One way would be to wrap all framebuffer methods in a `withParameters` call, although that is a little heavy (e.g. on the call stack). As an alternative, we could make a bindFramebuffer wrapper that just returns previous value from cache, without querying the context.

Reply:
> As an alternative, we could make a bindFramebuffer wrapper that just returns previous value from cache, without querying the context.

I like this idea. Also I was trying to add a unit tests to cover `_setDrawBuffers` and ran into issues, some of the code in this method needs to be handled by `polyfill`, requires more changes and a different PR. I will create an umbrella task to document all these.

Reply:
#322